Laboratoare/Lab/src/polimorfism/birdcontrollerex1/hierarchyex5/model/SportShoes.java

```diff
@@ -0,0 +1,33 @@
+package polimorfism.birdcontrollerex1.hierarchyex5.model;
+
+import polimorfism.birdcontrollerex1.hierarchyex5.model.Shoes;
+
+public class SportShoes extends Shoes {
+    private int kmNo;
+    private int rezistance = 100;
+

```

Comment:
de ce ai initializat aici rezistance? oricum va fi suprascrisa valoarea asta in constructor.